.github/workflows/build.yml

```
@@ -23,27 +22,34 @@ jobs:
           - python3
           - sed
         include:
```

```suggestion
        arch:
          - x86_64
          - aarch64
        include:
```

should fix CI. `include` doesn't like it when there's no overlap between itself and what's outside.